sci-libs/HDF5Plugin-Zstandard/HDF5Plugin-Zstandard-1.0_pre20190212.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR=ninja
+inherit cmake
+
+DESCRIPTION="ZSTD plugin for hdf5"
+HOMEPAGE="https://github.com/aparamon/HDF5Plugin-Zstandard"
+SRC_URI="https://github.com/aparamon/HDF5Plugin-Zstandard/archive/d5afdb5f04116d5c2d1a869dc9c7c0c72832b143.tar.gz -> HDF5Plugin-Zstandard.tar.gz"
+
```

Comment:
Add a line setting setting `S`
`S=${PN}-${COMMIT}`
This allows you to get rid of the `mv` in `src_prepare`.